docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md

```diff
@@ -5,12 +5,12 @@
 
 `ZITI_TIME_FORMAT=utc` - set the log message time format to UTC timestamp instead of milliseconds since start
 
-`ZITI_LOG=4` - set the log level of the underlying Ziti C SDK, higher is more verbose (level 4 means DEBUG)
+`ZITI_LOG=4` - set the log level of the underlying OpenZiti C SDK, higher is more verbose (level 4 means DEBUG)
 
 `TLSUV_DEBUG=4` - set the log level of the underlying libuv library, higher is more verbose (level 4 means DEBUG)
 
 <!-- Ken will update this when a better C SDK reference becomes available; this is a full URL because Docusaurus resolves relative and absolute URL and file paths at build time, and the Vercel build will always fail because it can't resolve the linked docs sites, e.g. CLANG doxygen site -->
-For more information about configuring the underlying Ziti C SDK with environment variables, see [the Ziti C SDK documentation](https://docs.openziti.io/docs/reference/developer/sdk/clang/).
+For more information about configuring the underlying OpenZiti C SDK with environment variables, see [the OpenZiti C SDK documentation](https://docs.openziti.io/docs/reference/developer/sdk/clang/).
```

Comment:
I updated this link to `[the Ziti C SDK documentation](../../../../docs/reference/developer/sdk/ziti-sdk-c)` recently and removed this comment. Let's just revert all these Ziti->OpenZiti. the repo is called "ziti c sdk". These should just be Ziti for better/worse

Reply:
OK, I think you're saying close this PR. I made these changes because you said something about preferring OpenZiti over Ziti in another PR for this same file. I didn't want to dismiss your approval on the other PR so I sent the requested changes here instead.

Reply:
https://github.com/openziti/ziti-doc/pull/582#pullrequestreview-1556825027